src/sonic-yang-models/yang-models/sonic-feature.yang

```diff
@@ -42,22 +42,22 @@ module sonic-feature{
                 leaf has_timer {
                     description "This configuration identicates if there is
                                 timer associated to this feature";
-                    type boolean;
-                    default false;
+                    type stypes:boolean_type;
```

Comment:
>boolean_type [](http://example.com/codeflow?start=32&length=12)

Add some test cases for the new type?

Reply:
Test cases for new boolean type were added. Please review.